src/tools/create-file-stasher.js

```diff
@@ -75,6 +75,11 @@ const createFileStasher = (input) => {
       return ZapierPromise.reject(new Error('rpc is not available'));
     }
 
+    const isRunningOnHydrator = _.get(input, '_zapier.event.method', '').indexOf('hydrators.') === 0;
+    if (!isRunningOnHydrator) {
+      return ZapierPromise.reject(new Error('Cannot stash files outside an hydration function/method.'));
```

Comment:
On my tests I've been using `appTester(App.hydrators.methodName)` to test file stashing/hydration, and that gets the `_zapier.event.method` filled as expected. Running the function directly will not work, unless they pass in a mock `z`.

I think this is fine because I don't expect developers to test the platform core in their apps directly, right?